src/parsing/Parse_metavariable_type.ml

```diff
@@ -0,0 +1,35 @@
+(* Copyright (C) 2019-2023 r2c
```

Comment:
You can put your name at the top of the file, and now it's
Semgrep Inc., not r2c.